logos/logos-orchestrator/tests/unit/main/test_request_logging.py

```
@@ -340,6 +345,383 @@ async def test_sync_response_async_job_success_logs_usage(monkeypatch):
+        },
+        61,
+        12,
+        10,
+        -1,
+        {"classified": True},
+        is_async_job=True,
+        request_path="v1/audio/translations",
+    )
+
+    assert result == {"status_code": 200, "data": {"text": "translated"}}
+    assert "stream" not in sent_params["payload"]
```

Pylint (4.0.6)

[error] Value 'sent_params' is unsubscriptable

(E1136)